lib/MongoDB/Collection.pm

```diff
@@ -727,6 +727,48 @@ sub find_one {
     )->execute->next;
 }
 
+=method find_id
```

Comment:
How should I go about this documentation? Right now it's copied almost verbatim from the `find_one` documentation. Would it be ok to just say it takes an `id` and to look at the `find_one` docs for more info?